typing/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,18 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Base application theme. -->
+    <style name="Theme.TypingAssist" parent="Theme.AppCompat.Light.NoActionBar">
+        <!-- Primary brand color. -->
+        <item name="colorPrimary">@color/purple_500</item>
```

Comment:
I guess the app theme should be consistant, we should discuss this with designers. Having different themes will make it difficult for us to introduce features like dark mode in future